workspaces/arborist/bin/reify.js

```diff
@@ -31,6 +33,13 @@ const printDiff = diff => {
 }
 
 module.exports = (options, time) => {
+  // Check for package.json
+  if (!fs.existsSync(path.join(options.path, 'package.json'))) {
+    log.error('No package.json found in the current directory.')
+    log.error('Please navigate to the correct directory or run npm init.')
+    return Promise.resolve('Aborted due to missing package.json')
```

Comment:
I think you meant Promise.reject? and then you'd need to wrap the message in a `new Error()`. 

But we could just make this `async` and throw properly too.

Reply:
I believe I have resolved this now.